core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -67,7 +67,7 @@
  */
 public class TabletsMetadata implements Iterable<TabletMetadata>, AutoCloseable {
 
-  private static class Builder implements TableRangeOptions, TableOptions, RangeOptions, Options {
+  public static class Builder implements TableRangeOptions, TableOptions, RangeOptions, Options {
```

Comment:
How about package private?

Reply:
@ivakegg I think this is the way to go. One tumbling block is crafting a unit test that is outside the ...metadata.schema package. A test created outside that package won't have access to the Builder methods. For the moment, I will push a version with a public Builder class that is tested using a mock table.